Example to avoid `useActor` when `send` is needed (#2480)
statelyai/xstate: docs/recipes/react.md

};
```

> If you need to use `send` in the same component to transition use `globalServices`

Reviewer: ```suggestion
If you need to send an event in the component that consumes a service, you can use the `service.send(...)` method directly:
```

```js
import React, { useContext } from 'react';
import { GlobalStateContext } from './globalState';
import { useSelector } from '@xstate/react';

const loggedInSelector = (state) => {
  return state.matches('loggedIn');
};

export const SomeComponent = (props) => {
  const globalServices = useContext(GlobalStateContext);
  const isLoggedIn = useSelector(globalServices.authService, loggedInSelector);

>   const send = globalServices.authService.send;

Reviewer: ```suggestion
  // Get `send()` method from a service
  const { send } = globalServices.authService;
```

> export const SomeComponent = (props) => {
>   const globalServices = useContext(GlobalStateContext);
>   const isLoggedIn = useSelector(globalServices.authService, loggedInSelector);
>   const send = globalServices.authService.send;
> 
>   return <>
>       {isLoggedIn ? 'Logged In' : 'Logged Out'}
>       <button onClick={() => send('TOGGLE')}>Toggle</button>
>    </>;
> };

Reviewer 2: Maybe something like this would demonstrate the pattern a little bit better? The problem with previous snippet was that a `TOGGLE` event was sent to the `authService` - which conceptually looked a little bit broken
```suggestion
export const SomeComponent = (props) => {
  const globalServices = useContext(GlobalStateContext);
  const isLoggedIn = useSelector(globalServices.authService, loggedInSelector);
  const send = globalServices.authService.send;

  return <>
      {isLoggedIn && <button type="button" onClick={() => send('LOG_OUT')}>Logout</button>}
   </>;
};
```

Author: You're totally right! I've push the commit with your sugestion
